evap/staff/importers.py

```diff
@@ -208,14 +208,20 @@ def _create_user_string(user):
         return "{} ({} {} {}, {})".format(user.username, user.title or "", user.first_name, user.last_name, user.email or "")
 
     @staticmethod
-    def _create_user_data_mismatch_warning(user, user_data):
-        return (mark_safe(_("The existing user would be overwritten with the following data:") +
+    def _create_user_data_mismatch_warning(user, user_data, test_run):
+        msg = "The existing user would be overwritten with the following data:" \
+            if test_run \
+            else "The existing user was overwritten with the following data:"
```

Comment:
i would prefer this as a regular if/else, since your syntax is intended for single line statements and you put that on multiple lines ;)

Reply:
:'(